frontend/components/auth/google_container.jsx

```diff
@@ -0,0 +1,15 @@
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router';
+//import { login, logout, signup, clearErrors } from '../../../actions/session_actions';
+import GoogleAuth from './google';
+
+const mapStateToProps = (state) => ({
+  clientId: state.clientId,
+  scopes: state.scopes
+});
+
+const mapDispatchToProps = (dispatch) => {
+  return null;
```

Comment:
why are we doing this? just remove the mapDispatchToProps if not needed

Reply:
Didn't implement this part yet, just stubbing it out but will probably add actions here for after login is successful